### PR TITLE
추천 컬렉션 피드 gql

### PR DESCRIPTION
**Pull request by robin-maki**

### CODE REVIEW

**Comment by robin-maki** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/withglyph/glyph/1677?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#1677** <a href="https://app.graphite.dev/github/pr/withglyph/glyph/1677?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#1676** <a href="https://app.graphite.dev/github/pr/withglyph/glyph/1676?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#1675** <a href="https://app.graphite.dev/github/pr/withglyph/glyph/1675?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @robin-maki and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/penxle.com/scripts/create-search-index.ts [hunk 1]
```diff
@@ -114,6 +114,44 @@ const indexData: Record<string, Omit<IndicesCreateRequest, 'index'>> = {
...
+      properties: {
+        name: { type: 'text', analyzer: 'ngram_2' },
+        space: {
+          properties: {
+            id: { type: 'keyword' },
+          },
+        },
+        reputation: { type: 'rank_feature' },
+        hasThumbnail: { type: 'boolean' },
+        lastPostPublishedAt: { type: 'date', format: 'epoch_millis' },
```
Comment (devunt) [inline apps/penxle.com/scripts/create-search-index.ts:151]: P3: `epoch_millis` 여기서는 안 빼도 되는건가여

File: apps/penxle.com/src/lib/server/jobs/search.ts [hunk 2]
```diff
@@ -99,3 +108,69 @@ export const IndexAllPostsInSpaceJob = defineJob('indexAllPostsInSpace', async (
...
+
+  await elasticSearch.index({
+    index: indexName('collections'),
+    id: collection.id,
+    document: {
+      name: collection.name,
+      space: {
+        id: collection.spaceId,
+      },
+      reputation: await calculatePostReputation(collection.id),
```
Comment (devunt) [inline apps/penxle.com/src/lib/server/jobs/search.ts:171]: P1: `calculateCollectionReputation` 이어야 하는거겠져?

File: apps/penxle.com/src/lib/server/utils/post.ts [hunk 3]
```diff
@@ -118,6 +118,21 @@ export const calculatePostReputation = (postId: string) =>
     600,
   );
 
+export const calcurateCollectionReputation = async (collectionId: string) =>
```
Comment (devunt) [inline apps/penxle.com/src/lib/server/utils/post.ts:121]: P2: `calculate`

**Review by devunt — CHANGES_REQUESTED** [review]

**Review by devunt — APPROVED** [review]

**Pull request merged**